### PR TITLE
Sentry 47011398

### PR DESCRIPTION
**Pull request by matthew-andrews**
and various upgrades

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -40,14 +41,14 @@ app.use(allowCrossDomain);
...
-		return response.json();
-	})
-	.then(function(json) {
-		res.send(json);
-	})
-	.catch(next);
+		.then(function(response) {
+			res.status(response.status);
+			return response.text()
+				.then(function(text) {
```
Comment (matthew-andrews) [inline app.js]: this was the bug caught in sentry…  `json` will error if not json, but if use `text` it'll blindly pass through

File: package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "next-markets-proxy-api",
+  "name": "ft-next-markets-proxy-api",
```
Comment (matthew-andrews) [inline package.json]: next build tools

File: package.json [hunk 3]
```diff
@@ -8,15 +8,17 @@
     "express": "^4.8.x",
     "express-errors-handler": "^1.0.0",
     "forever": "^0.11.1",
-    "haikro": "^1.6.4",
-    "isomorphic-fetch": "^1.0.0",
-    "nock": "^0.51.0",
```
Comment (matthew-andrews) [inline package.json]: not a dep, a dev dep

File: test/apiSpec.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+var fixtures = {
+	quotes: fs.readFileSync('test/fixtures/quotes.json'),
+	unknown: fs.readFileSync('test/fixtures/unknown.json')
+};
+
+describe('API proxy', function() {
+
+	it('responds with json for good requests', function() {
+		nock(host).get(urls.quotes + 'test').reply(200, fixtures.quotes);
+		return fetch('http://localhost:' + port + '/securities/v1/quotes?symbols=test')
```
Comment (matthew-andrews) [inline test/apiSpec.js]: mocha understands promises, so don't use `done`

**Comment by adgad** [comment]
:+1:

**Comment by adgad** [comment]
smashing!

**Pull request merged**